optimize-nextflow.py

```diff
@@ -240,12 +268,21 @@ def generate_single(process_metrics: ProcessMetrics) -> str:
         """
         mem_optimal = process_metrics.optimize_mem()
         cpu_optimal = process_metrics.optimize_cpu()
+
+        mem_optimal_fmt = repr([f"{mem}.GB" for mem in mem_optimal])
```

Comment:
I think this is a nice usage of `repr` as it captures what you're trying to do. 

Just as a note: I'm not sure lists implement a custom `__str__()` function, so `str()` is technically the same as `repr()`

```
str(["1.GB", "2.GB"]) == repr(["1.GB", "2.GB"])
True
```

Reply:
Haha, I also checked whether there's a difference between `str()` and `repr()`. I could extend the `list` class with an alternative `__repr()__` method, but I think a single `replace()` call is fine. If the list format was more complicated, it would make sense to define a modified `list` subclass.